lib/Controller/PageController.php

```diff
@@ -122,21 +126,36 @@ public function indexFPath(): TemplateResponse {
 	 */
 	#[NoAdminRequired]
 	#[NoCSRFRequired]
+	#[RequireSetupOk]
```

Comment:
Why not main template here also ?

Reply:
What about main template?
Can you explain more about this?

Reply:
Like this: `#[RequireSetupOk(template: 'main')]`